### ISSUE TITLE
Add troubleshooting for "AADSTS50058"

### ISSUE THREAD

**Issue by skaue** [MicrosoftDocs/dynamics-365-unified-operations-public#716]
Sometimes users are unable to authenticate. They keep getting an error stating they could not login. They will need to sign out in the add-in, reload the add-in and re-authenticate. When the dialog pops up and asks for credentials, don't pick the user from selection, but manually authenticate by selecting "other user". Type in username and password to force a clean authentication. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3fcd3b1a-d14e-7738-b4c3-c4aac5a9c791
* Version Independent ID: 85742fde-3ab0-ac8d-77c5-929a53f4a08f
* Content: [Troubleshoot the Office integration - Finance & Operations | Dynamics 365 | #MSDyn365FO](https://docs.microsoft.com/en-us/dynamics365/unified-operations/dev-itpro/office-integration/office-integration-troubleshooting?cid=kerryherger)
* Content Source: [articles/dev-itpro/office-integration/office-integration-troubleshooting.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/live/articles/dev-itpro/office-integration/office-integration-troubleshooting.md)
* Service: **dynamics-ax-platform**
* GitHub Login: @ChrisGarty
* Microsoft Alias: **cgarty**

**Comment by ShylaThompson** [comment]
@ChrisGarty  can you review and update the topic if necessary?

**Comment by skaue** [comment]
I had a issue for this today. It was not enough to simply sign out and reload the app. When the authenticate dialog prompted, I had to force the user not to select himself, but choose "use another login" (don't recall the exact text), where he entered username and password manually. Then he was able to fully re-authenticate and get a good login. Otherwise, login failed constantly. We tried against different AADs (customers), and the same error.
